src/main/java/seedu/eduke8/question/QuestionList.java

```diff
@@ -4,88 +4,51 @@
 import seedu.eduke8.common.DisplayableList;
 
 import java.util.ArrayList;
-import java.util.Random;
 
 public class QuestionList implements DisplayableList {
-    private ArrayList<Displayable> allQuestions;  // list of questions for the particular topic
-    private ArrayList<Question> quizQuestions;
-    private int currentQuestionNumber;
 
-    private static final Random RANDOM = new Random();
+    private ArrayList<Displayable> questions;  // list of questions for the particular topic
 
     public QuestionList(ArrayList<Displayable> questions) {
-        allQuestions = questions;
-        quizQuestions = null;
-        currentQuestionNumber = 0;
+        this.questions = questions;
     }
 
-    public void setQuizQuestions(int numberOfQuestionsForQuiz) {
-        int numberOfQuestionsSelected = 0;
-        // prevent repeated questions from being selected again
-        ArrayList<Integer> integersChosen = new ArrayList<>();
+    /*
+    public QuizQuestionsManager getQuizQuestionList(int numberOfQuestionsForQuiz) throws Eduke8Exception {
+        QuizQuestionsManager quizQuestionsManager = new QuizQuestionsManager();
```

Comment:
Since you're getting QuizQuestionList with this function, would calling the new QuizQuestionsManager object quizQuestionList instead of quizQuestionsManager?

Reply:
Oh! Sorry! This comment is supposed to be deleted from the commit, it was from a previous version! I will change it now.